mxd/MXD_dataexchange_tutorial1.md

```diff
@@ -0,0 +1,151 @@
+    "@id": "{{POLICY_ID}}",
+    "policy": {
+        "@type": "Policy",
+        "odrl:permission": [
+            {
+                "odrl:action": "USE",
+                "odrl:constraint": {
+                    "@type": "LogicalConstraint",
+                    "odrl:and":
+                        {
+                            "@type": "Constraint",
+                            "odrl:leftOperand": "BusinessPartnerNumber",
+                            "odrl:operator": {
+                                "@id": "odrl:eq"
+                            },
+                            "odrl:rightOperand": "{{BPN123}}"
+                        },
+                }
+            }
+        ]
+    }
+} 
+```
+
+Bob tells Alice that he has created the right policy. Let´s see if Alice can now find the data asset. Execute the request again using the following curl command:
+
+```shell
+curl -G -X GET "${ALICE_DATAMGMT_URL}/data/catalog" \
+    --data-urlencode "providerUrl=${BOB_IDS_URL}/api/v1/ids/data" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    -s | jq
+```
+
+Let´s see if Alice can see the Asset. Can you find it?
+
+Once again Alice cannot find the data offer. This is by design and to be expected since Bob has only created an asset and a policy definition. An asset and a policy cannot be displayed to Alice as a consumer without a contract definition.
+**This is the first lesson for this tutorial: A contract must be defined between two parties that want to exchange data. This contract must always contain an asset and a policy.**
+
+Add image showing that a conctract defiition consists of a data asset and a policy definition
+
+Action (Bob): Create a contract definition including the asset and the policy you have created. For this, use the following curl command:
+
+```shell
+curl -X POST "${BOB_DATAMGMT_URL}/data/contractdefinitions" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    --data '{
+                "id": "1",
+                "criteria": [
+                    {
+                        "operandLeft": "asset:prop:id",
+                        "operator": "=",
+                        "operandRight": "1"
+                    }
+                ],
+                "accessPolicyId": "1",
+            }' \
+    -s -o /dev/null -w 'Response Code: %{http_code}\n'
+```
+
+Let´s see if Alice can finally see the Asset.
+action (Alice): Execute the request again using the following curl command:
+
+```shell
+curl -G -X GET "${ALICE_DATAMGMT_URL}/data/catalog" \
+    --data-urlencode "providerUrl=${BOB_IDS_URL}/api/v1/ids/data" \
+    --header 'X-Api-Key: password' \
+    --header 'Content-Type: application/json' \
+    -s | jq
+```
+
+Finally Alice can see the Contract Offer from Bob.
+Congratulations on yor first successful data exchange in your own data sapce!
+
+Before you start the next tutorial please delete all data:
```

Comment:
We can remove this part. We don't have a shortcut command to clear all the data. Although we can provide multiple curls to delete it.
We will use new ids so it will not harm.